src/state/slices/common-selectors.ts

```diff
@@ -85,3 +89,57 @@ export const selectPortfolioFiatBalances = createDeepEqualOutputSelector(
       return acc
     }, {}),
 )
+
+export const selectPortfolioFiatBalancesByAccount = createDeepEqualOutputSelector(
+  selectAssets,
+  selectPortfolioAccountBalancesBaseUnit,
+  selectMarketDataSortedByMarketCap,
+  (assetsById, accounts, marketData) => {
+    return Object.entries(accounts).reduce(
+      (acc, [accountId, balanceObj]) => {
+        acc[accountId] = Object.entries(balanceObj).reduce(
+          (acc, [assetId, cryptoBalance]) => {
+            const asset = assetsById[assetId]
+            if (!asset) return acc
+            const precision = asset.precision
+            const price = marketData[assetId]?.price ?? 0
+            const cryptoValue = fromBaseUnit(bnOrZero(cryptoBalance), precision)
+            const fiatBalance = bnOrZero(bn(cryptoValue).times(price)).toFixed(2)
+            acc[assetId] = fiatBalance
+
+            return acc
+          },
+          { ...balanceObj },
+        )
+
+        return acc
+      },
+      { ...accounts },
+    )
+  },
+)
+
+export const selectSortedAssets = createSelector(
```

Comment:
```suggestion
export const selectSortedAssets = createDeepEqualOutputSelector(
```

anything that returns a non-primitive should be a deep equal output selector - the output can be the same by value but not by ref, and component renders are way more expensive than a deep equal comparison